pygmt/tests/test_clib_vectors_to_arrays.py

```diff
@@ -80,6 +80,19 @@ def test_vectors_to_arrays_pandas_nan():
     _check_arrays(arrays)
 
 
+def test_vectors_to_arrays_pandas_string():
+    """
+    Test the vectors_to_arrays function with pandas Series containing datetime64.
+    """
+    vectors = [
+        pd.Series(["abc", "defhig"]),
+        pd.Series(["abcdef", "123456"], dtype="string"),
```

Comment:
There are 3 'string' dtypes in pandas now according to https://pandas.pydata.org/pandas-docs/version/2.2/reference/api/pandas.StringDtype.html:

- `string` or `string[python]` (default as of Pandas 2.x)
- `string[pyarrow]`
- `string[pyarrow_numpy]`

I think the expectation is that in pandas 3.0, the default StringDtype will change from `string[python]` to `string[pyarrow]` (see https://pandas.pydata.org/pdeps/0010-required-pyarrow-dependency.html). So maybe we should explicitly test for `string[python]` here? Or maybe test for all three variants.

Reply:
Done in d8777e5e8f4878a463a4524f65033c95a1352dc6.